docs/source/user/interface.rst

```diff
@@ -88,13 +88,13 @@ The same information is also available in the Tabs menu:
 .. image:: images/interface_tabs_menu.png
    :align: center
 
-It is often useful to focus on a single document or activity without
-closing other tabs in the main area. Single-Document Mode toggles the
-view of the main area to show only a single tab at a time:
+It is often useful to focus on a single document or activity without closing
+other tabs in the main area. The Single-Document Mode item in the View menu
```

Comment:
IMHO I prefer the former simply because "The Single-Document Mode item in the View menu" seems wordy to me

Reply:
Hmm, you're right, it is wordy. I was trying to clearly indicate where to find the Single-Document Mode menu item. I'll try this again.

Reply:
I would probably separate the description of the feature (single-document mode) from its visual representation. Something like a second sentence of "Single-document mode accomplishes this..... It can be enabled in the View menu...